daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -58,6 +70,41 @@ To terminate your workflow with an ID `12345678`, run:
 POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/terminate
 ```
 
+### Raise an event
+
+Run the following for your workflow to create "wait for external event" tasks that:
+
+1. Subscribe to external events.
+1. Await those tasks to block execution until the event is received.
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceID>/raiseEvent/<eventName>
+```
+
+> An `eventName` can be any function. 
+
+### Pause or resume a workflow
+
+To plan for down-time, wait for inputs, and more, you can pause and then resume a workflow. To stop a workflow with an ID `12345678` until triggered to resume, run:
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/pause
+```
+
+To resume a workflow with an ID `12345678`, run:
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/resume
+```
+
+### Purge a workflow 
+
+To remove actor state from your state store and avoid overloading your state stores, run:
```

Comment:
Actors are an implementation detail and I suggest just saying remove the state. 
```suggestion
To remove all the workflow and activity state from your state store to reduce storage use, run:
```
Also when should you call purge? What is the recommendation, once a workflow has completed? SInce there is a danger here that if you call this whilst a workflow is running, won't this invalidate the workflow? What is the guidance here? Is there a "Danger: Be careful when calling purge API" warning?

Reply:
@RyanLettieri could you elaborate the risks of calling purge?

Reply:
If the current runtime state is not completed, it is not able to be purged and will return an error stating "orchestration has not yet completed". Purge should be called on completed workflow instances to free up potentially non-needed items in the statestore.

Reply:
I added a comment to add this detail